### PR TITLE
[PR-14564] Add points to filter documentation

### PR DESCRIPTION
**Pull request by ymasiuk**
Related to https://github.com/talkable/talkable/pull/6977

## Related Stories

[](https://talkable.atlassian.net/browse/PR-14564)

### CODE REVIEW

**Comment by zhuravel** [comment]
@ymasiuk Please don’t add demo link until deployed to void and there’s a demo.

File: source/campaigns/editor/filters.rst [hunk 1]
```diff
@@ -406,6 +406,43 @@ Available options:
...
+    ============================================================== ==============
+    ``{{ "10050" | points: delimiter: "_" }}``                     ``100_500``
+    -------------------------------------------------------------- --------------
+    ``{{ "100" | points: format: "%nP" }}``                        ``1000P``
+    -------------------------------------------------------------- --------------
+    ``{{ "100" | points: convert: false }}``                       ``$100``
+    ``{{ "1000" | points: convert: false, delimiter: "," }}``      ``$1,000``
+    -------------------------------------------------------------- --------------
+    ``{{ "100" | points: percentage_discount: true }}``            ``500``
+    ============================================================== ==============
```
Comment (zhuravel) [inline source/campaigns/editor/filters.rst:442]: Corrected indentation and removed blank lines:
```suggestion
points
......

Formats number using conversion rate for loyalty from site settings.

.. code-block:: liquid

   {{ "50" | points }}

Returns ``500``.

Available options:

* ``delimiter``
* ``format``
* ``convert (boolean)``
* ``percentage_discount (boolean)``

.. container:: ptable

   ============================================================== ==============
   Example                                                        Result
   ============================================================== ==============
   ``{{ "10050" | points: delimiter: "_" }}``                     ``100_500``
   -------------------------------------------------------------- --------------
   ``{{ "100" | points: format: "%nP" }}``                        ``1000P``
   -------------------------------------------------------------- --------------
   ``{{ "100" | points: convert: false }}``                       ``$100``
   ``{{ "1000" | points: convert: false, delimiter: "," }}``      ``$1,000``
   -------------------------------------------------------------- --------------
   ``{{ "100" | points: percentage_discount: true }}``            ``500``
   ============================================================== ==============
```

**Review by zhuravel — COMMENTED** [review]
If this filter formats money when `convert: false` is passed, why doesn’t it support other options from `money` filter?

* unit
* separator
* precision (integer)
* strip_insignificant_zeros (boolean)

cc @necroua

**Review by necroua — APPROVED** [review]
LGTM

**Pull request merged**